Frends.AzureBlobStorage.ReadBlob/CHANGELOG.md

```diff
@@ -2,4 +2,10 @@
 
 ## [1.0.0] - 2022-04-08
 ### Added
-- Initial implementation
\ No newline at end of file
+- Initial implementation
+
+## [1.0.0] - 2022-04-21
```

Comment:
You don't need to update the changelog, since the original version is not yet released.